.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
https://eslint.org/docs/user-guide/configuring#configuration-file-formats
`.eslintrc` File format was deprecated

Reply:
Oh I didn't know about this.. 😄 I fixed it on commit https://github.com/project-dobby/dobby/pull/11/commits/63174bc4b9cc8b71a7b954c6796093135272ac5b